dev/com.ibm.ws.jsp/src/com/ibm/wsspi/jsp/taglib/config/TldPathConfig.java

```diff
@@ -74,4 +74,46 @@ public void setUri(String string) {
     public boolean containsListenerDefs() {
         return containsListenerDefs;
     }
+
+    public boolean isCustomURIUsed() {
+        return this.forceCustomURI;
+    }
+
+    /*
+     * Ignore the URI set in the TLD attribute and used the URI 
```

Comment:
1) What TLD attribute?
2) `and used` should be `and use`
3) Should we link to the TldPathConfig constructor for the javadoc.